api/session/auth_session.py

```diff
@@ -68,8 +66,7 @@ def create_session_details_with_token(cls, account_id: str):
             "accountId": account_id,
             "iat": int(datetime.now().timestamp()),
             "exp": int(
-                datetime.now().timestamp()
-                + env.config.get("FSD_SESSION_TIMEOUT_SECS", 0)
+                datetime.now().timestamp() + Config.FSD_SESSION_TIMEOUT_SECS
```

Comment:
Should FSD_SESSION_TIMEOUT_SECS=0 be added to DefaultConfig so that this works (or has 0 as a default)?

Reply:
It has the following default: FSD_SESSION_TIMEOUT_SECS = 3600  # 1 hour